poly_borders/src/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+project(poly_borders_postprocessor_library)
```

Comment:
Why do you need a separate library here?
Can't we structure this as all our other binaries (e.g. generator_tool here)?
```
poly_borders/
  borders_data.cpp
  borders_data.hpp
  help_structures.cpp
  help_structures.hpp

  poly_borders_tests/
    ...
```